lib/bin/lwm2m_carrier/Kconfig

```diff
@@ -26,6 +26,8 @@ menuconfig LWM2M_CARRIER
 	depends on DOWNLOAD_CLIENT && DOWNLOAD_CLIENT_TLS
 	# Credentials NCS
 	depends on MODEM_KEY_MGMT
+	# GPS
+	depends on !NRF9160_GPS_HANDLE_MODEM_CONFIGURATION
```

Comment:
I'm not sure we should have this dependency in the lib. I think it should rather be that `NRF9160_GPS_HANDLE_MODEM_CONFIGURATION` that `depends on !LWM2M_CARRIER`

Reply:
yes i wasn't sure where to put it, so it ended up here with all the other dependencies 🙃